src/main/kotlin/no/nav/familie/tilbake/behandlingskontroll/BehandlingsstegMetaData.kt

```diff
@@ -0,0 +1,7 @@
+package no.nav.familie.tilbake.behandlingskontroll
+
+import no.nav.familie.tilbake.behandlingskontroll.domain.Behandlingssteg
+import no.nav.familie.tilbake.behandlingskontroll.domain.Behandlingsstegstatus
+
+data class BehandlingsstegMetaData(val behandlingssteg: Behandlingssteg,
```

Comment:
Litt usikker på om jeg synes metadata er riktig her. Status kan knakje kalles metadata for behandlingssteg, men behandlingssteg er ikke metatdata for behandlingssted. BehandlingsstegMedStatus? Jeg vet ikke helt hva som vil være et godt navn her.